addons/account_edi_ubl_cii/models/account_move_send.py

```diff
@@ -145,7 +145,8 @@ def _hook_invoice_document_after_pdf_report_render(self, invoice, invoice_data):
             return
 
         # Read pdf content.
-        reader_buffer = io.BytesIO(invoice_data['pdf_attachment_values']['raw'])
+        pdf_values = invoice_data.get('pdf_attachment_values') or invoice_data['proforma_pdf_attachment_values']
```

Comment:
Why? In case of error, you don't enter this hook so `proforma_pdf_attachment_values` can't be there :thinking:

Reply:
As discussed you do enter the hook from generate_invoice_fallback :P